proposals/lip-0052.md

```diff
@@ -1680,20 +1763,26 @@ def unlock(
     nftID: NFTID
 ) -> None:
 
-    if NFTStore[nftID] does not exist:
-        emitFailedUnlockEvent(module, nftID, RESULT_NFT_DOES_NOT_EXIST)
-        raise Exception("NFT substore entry does not exist")
+    nft = getNFT(nftID)
```

Comment:
I think we should handle the possible exception from this function by emitting an event with `RESULT_NFT_DOES_NOT_EXIST` if the call to `getNFT` fails.

Reply:
Done in [this commit](https://github.com/LiskHQ/lips/pull/477/commits/876ae2c110612053e85a0c54fb65574a7e1b4bc1).